src/test/java/com/google/api/codegen/testdata/nodejs_doc_message_library.baseline

```diff
@@ -999,9 +1001,13 @@ var Book = {
    * @enum {number}
    */
   Rating: {
-    GOOD: 0,
+    /**
+     * GOOD enum description
+     */  GOOD: 0,
```

Comment:
Probably line breaking rule has been wrong (or at least weird) here, could you make a fix on the new template?  There should be a linebreak between the comment and the definition, like
```javascript
/**
 * GOOD enum description
 */
GOOD: 0,
```